taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    description: Sends Slack message to release testers that Testrail Milestone was created.
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 120
+        env:
+            TESTRAIL_PROJECT_ID: '59' # Fenix Browser
+            TESTRAIL_TEST_SUITE_ID: '49319' # Test Automation Release Milestone
+    run:
+        use-caches: false
+        using: run-commands
+        secrets:
+            - name: project/mobile/ci/testrail
+              key: testrailCredentials
+              path: .testrail_credentials.json
+              json: true
+    routes:
+        - notify.slack-channel.G016BC5FUHJ.on-failed
+    scopes:
+        - queue:route:notify.slack-channel.G016BC5FUHJ # notify mobile-alerts-sandbox on failure
```

Comment:
should this be `queue:route:notify.slack-channel.G016BC5FUHJ.on-failed`?

Reply:
good question! no really sure.

In `testrail_main.py` i'm using the `Taskcluster API` to handle the cases of `failed` and `succeed`, and I'm not exactly sure what `queue:route` is used for, but I didn't want a potential conflict so I didn't include it.

That said, I'm not sure what impact, if any, it has by including it or leaving it out?

All that to say, it was intentional, though that isn't to say it's correct.